src/scene.js

```diff
@@ -78,11 +80,8 @@ export default function Scene(source, config_source, options) {
     log.setLevel(this.logLevel);
 }
 
-Scene.create = function ({source, config}, options = {}) {
-    if (!(source instanceof TileSource)) {
-        source = TileSource.create(source);
-    }
-    return new Scene(source, config, options);
+Scene.create = function ({config}, options = {}) {
```

Comment:
Actually is this function still needed/desirable if it has no extra functionality?